plugins/backend-serverless-app/readme.md

```diff
@@ -0,0 +1,27 @@
+# @dotcom-tool-kit/backend-serverless-app
+
+A bootstrap plugin that provides the minimum required Tool Kit plugins for a "backend" (aka an [API](https://github.com/Financial-Times/next/wiki/Naming-Conventions#apis)) that is deployed to AWS. The plugins are:
+
+- [`@dotcom-tool-kit/npm`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/npm)
+- [`@dotcom-tool-kit/circleci-deploy`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/circleci-deploy)
+- [`@dotcom-tool-kit/serverless`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/serverless)
+- [`@dotcom-tool-kit/node`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/node)
+- [`@dotcom-tool-kit/husky-npm`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/husky-npm)
+- [`@dotcom-tool-kit/secret-squirrel`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/secret-squirrel)
+
+This bootstrap plugin is also preconfigured to run the `ServerlessRun` task on the hook `run:local`, and binds the tasks defined by the `serverless` plugin to the hooks defined by `circleci-deploy`.
+
+## Installation
```

Comment:
**chore:** we should add instructions to provide the serverless options too.

Reply:
Isn't that already covered in the [root README](https://github.com/Financial-Times/dotcom-tool-kit#configuration)

Reply:
For the bootstrap plugins, it won't be clear that the individual plugins it contains will need to be configured so we should add these instructions.

Reply:
We have added this work to https://financialtimes.atlassian.net/browse/CPP-1355